strapi/strapi
# [Marketplace] Add submit providers button

### What does it do?

Updates the button on the marketplace page heading so that it can handle plugins and providers

### Why is it needed?

To point the user to the correct form for submitting plugins and providers

### How to test it?

- Go to the marketplace page
- The default tab is plugins, you should see submit plugin, click submit plugin, you should be on the submit plugin form
- Click the providers tab, you should see submit provier, click submit provider, you should be on the submit provider form
